app/src/main/java/de/westnordost/streetcomplete/quests/fire_hydrant/AddFireHydrantDiameter.kt

```diff
@@ -29,7 +32,16 @@ class AddFireHydrantDiameter : OsmFilterQuestType<FireHydrantDiameterAnswer>() {
 
     override val enabledInCountries = NoCountriesExcept("DE","BE","GB","PL","IE","FI","NL")
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_fireHydrant_diameter_title
+    override fun getTitleArgs(tags: Map<String, String>, featureName: Lazy<String?>): Array<String> =
+        arrayOfNotNull(getNameOrBrandOrOperatorOrRef(tags))
```

Comment:
so fire hydrant with name, bran of operator will show up like this?

> What diameter is specified on the sign for this fire hydrant? (reference number: Random City Fire Hydrant Maintenance Department)

I think that getRef function would be better (maybe it should be created)

Reply:
done